db/schema.rb

```diff
@@ -14,7 +14,6 @@
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "citext"
-  enable_extension "pg_stat_statements"
```

Comment:
We don't want to remove this.

Reply:
I think sneaks in because of differences between my local Postgres and prod (at least that's my guess)